rs/ledger_suite/icp/ledger/tests/tests.rs

```diff
@@ -1720,6 +1720,61 @@ fn test_icrc21_standard() {
     ic_ledger_suite_state_machine_tests::test_icrc21_standard(ledger_wasm(), encode_init_args);
 }
 
+/// Arguments taken by the account_balance candid endpoint.
+#[derive(Clone, Eq, PartialEq, Hash, Debug, CandidType)]
+struct AccountIdentifierBlob {
+    account: ByteBuf,
+}
+
+#[test]
+fn test_account_balance_non_standard_account_identifier_length() {
+    let env = StateMachine::new();
+    let payload = LedgerCanisterInitPayload::builder()
+        .minting_account(MINTER.into())
+        .build()
+        .unwrap();
+    let canister_id = env
+        .install_canister(ledger_wasm(), Encode!(&payload).unwrap(), None)
+        .expect("Unable to install the Ledger canister with the new init");
+
+    // account balance of account identifier of correct length
+    let p1 = PrincipalId::new_user_test_id(1);
+    let valid_account_identifier_bytes = ByteBuf::from(AccountIdentifier::from(p1.0).to_vec());
+    assert_eq!(valid_account_identifier_bytes.len(), 32);
+    let res = Decode!(
+        &env.execute_ingress(
+            canister_id,
+            "account_balance",
+            Encode!(&AccountIdentifierBlob {
+                account: valid_account_identifier_bytes
+            })
+            .unwrap()
+        )
+        .expect("failed to query blocks")
```

Comment:
```suggestion
        .expect("failed to query account_balance")
```

Reply:
Thanks! I fixed this one, and the same one a few lines below.